news/migrations/0002_delete_subscriber.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('news', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='Subscriber',
```

Comment:
I recommend moving this to a separate commit, to be deployed after the removal from `news/models.py`.